foundry/foundry.py

```diff
@@ -498,3 +512,41 @@ def download_file(file):
             print(results)
 
         return self
+
+    def build(self, spec, globus=False, interval=3, file=False):
+        """Build a Foundry Data Package
+        Args:
+            spec (multiple): dict or str (relative filename) of the data package specification
+            globus (bool): if True use Globus to fetch datasets
+            interval (int): Polling interval on checking task status in seconds.
+            type (str): One of "file" or None
+
+        Returns
+        -------
+        (Foundry): self: for chaining
+        """
+
+        print("Building Data Package")
+        num_cores = multiprocessing.cpu_count()
+
+        def start_download(ds, interval=interval, globus=False):
+            print("=== Fetching Data Package {} ===".format(ds.name))
+            f = Foundry().load(ds.name, download=False)
+            f = f.download(interval=interval, globus=globus)
+            return {"success": True}
+
+        if file:
+            with open(file, "r") as fp:
+                fs = FoundrySpecification(**json.load(fp))
+        else:
+            fs = FoundrySpecification(**spec)
+
+        fs.remove_duplicate_dependencies()
+
+        results = Parallel(n_jobs=num_cores)(
```

Comment:
what does this do? can't tell from context, could you please add an inline comment?